components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -231,6 +231,17 @@ class BrowserToolbar @JvmOverloads constructor(
             editToolbar.urlView.setTextColor(value)
         }
 
+    /**
+     * Sets the colour of the tracking protection icon.
+     */
+    var trackingProtectionColor: Int? = null
+        set(value) {
+            value?.let {
```

Comment:
We should probably just default to transparent instead of making this nullable.

Reply:
We can do that but it will not reflect the actual color of the icon. I coudn't find a easy way to get the color from a `drawable` (`displayToolbar.trackingProtectionIconView.background`), I'm not sure if `displayToolbar.trackingProtectionIconView.solidColor` provides us the color of the icon, as the documentation is not very descriptive https://developer.android.com/reference/android/view/View#getSolidColor() .

Reply:
Can the colour just come from a resource and/or use the defaultColor to remove the nullability? We do [something similar](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt#L144-L148) already with defaultColor.

Reply:
That could be an alternative!

Reply:
Updated!